UC-Berkeley-Extension/global
# Sass compile gulp

## Description

- Added gulp and browsersync that can compile & serve css and js changes.

## Testing Instructions

- Navigate to `/web/themes/custom/bglobal` dir
  - Note: If you already have `node_modules` dir from previous `npm install`, remove this dir so it can have fresh dependencies.
- `npm install`
- `npm start`
- Check that styleguide is served on localhost
- In your IDE, navigate to `web/themes/custom/bglobal/source/_patterns/00-atoms/buttons/_buttons.scss` file. Change the background color to something.
- On your terminal, check that file watch is working, and that it auto-compiled the changes.
- Check your styleguide that was served - it should have auto-refreshed with the new changes, and changed the button color.
- For a preview of the functionality, check [this link](https://drive.google.com/file/d/1HbTtxrNiq6GS72YodmgnWXIHhunP7v7X/view?usp=sharing).

## Note
- PR #3 can be removed, if this gets merged in.
